android/res/layout/place_page_details.xml

```diff
@@ -42,7 +42,7 @@
 
     <include layout="@layout/place_page_entrance"/>
 
-    <include layout="@layout/place_page_phone"/>
+    <include layout="@layout/place_page_phone_list"/>
```

Comment:
nit:  As we already will have in this big vertical linear layout duplicating layout items, it would be great, if you will be able to refactor it to RecyclerView.